android-sdk/src/main/java/org/sagebionetworks/bridge/android/manager/AuthenticationManager.java

```diff
@@ -269,35 +251,26 @@ public UserSessionInfo getUserSessionInfo() {
         // TODO: a way to distinguish if session is null because we haven't signed on, or if it
         // was invalidated
         UserSessionInfo session = apiClientProvider.getUserSessionInfoProvider()
```

Comment:
My intention is to expose two ways to get the UserSessionInfo, one for synchronous use (returns from accountDao only) and one async which may make a network call (and has to be done on the background thread). 

The devs should be using the Manager, and shouldn't have a way of getting at apiClientProvider or need to know  details of how UserSessionInfo is handled in rest-client. The only people who need to know how all that works are people contributing to rest-client and android-sdk.

I definitely want to expose only one way (actually two, sync and async) to get UserSessionInfo, StudyParticipant, Consents, so let me know if I missed anything.